### PR TITLE
[analytics] fixed anonymous ID trait in identify()

### PR DESCRIPTION
**Pull request by jakobhero**
In order to attribute client side events to signed up users, I changed the anonymous ID trait on server side identify calls to the one used on the client side.
/werft analytics=segment|TEZnsG4QbLSxLfHfNieLYGF4cDwyFWoe

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/4891#" title="Author self-approved">jakobhero</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **jankeromnes** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @jankeromnes` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/4891#" title="Author self-approved">jakobhero</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **svenefftinge** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @svenefftinge` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**
- **[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: components/server/src/auth/login-completion-handler.ts
Comment (svenefftinge) [inline components/server/src/auth/login-completion-handler.ts:88]: we need a guard for undefined in case the cookie wasn't set.

**Review by svenefftinge — APPROVED** [review]

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: cad8b26a814a179c9b58bd794906d74ce4ce4400</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/4891#" title="Author self-approved">jakobhero</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/4891#pullrequestreview-712640353" title="Approved">svenefftinge</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [svenefftinge]
- ~~[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)~~ [svenefftinge]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: components/server/src/auth/login-completion-handler.ts
Comment (svenefftinge) [inline components/server/src/auth/login-completion-handler.ts:86]: nit: please use `const` if the value never changes

File: components/server/src/auth/login-completion-handler.ts
Comment (svenefftinge) [inline components/server/src/auth/login-completion-handler.ts:88]: nit: please use brackets and newline indent

**Pull request merged**
